solr/test-framework/src/java/org/apache/solr/cloud/AbstractFullDistribZkTestBase.java

```diff
@@ -147,6 +147,8 @@ public void beforeTest() {
   protected volatile ChaosMonkey chaosMonkey;
 
   protected Map<String, CloudJettyRunner> shardToLeaderJetty = new ConcurrentHashMap<>();
+  protected volatile Map<String, CloudSolrClient> solrClientForCollectionCache =
```

Comment:
umm...   i thought maybe becasue multiple things reading?   i can change.

Reply:
`volatile` has to do with ensuring threads can read it if it is written to by another thread.  This refers to the field itself, not what may be inside (i.e. it is to the actual field reference to CHM, not what's *in* CHM).  Since you set a value at the declaration and never set it anywhere else, just remove "volatile" and make it final as well to communicate this fact.